fitbenchmarking/core/fitting_benchmarking.py

```diff
@@ -375,29 +374,14 @@ def loop_over_minimizers(controller, minimizers, options, grabbed_output):
             ########################
             # Loops over Jacobians #
             ########################
-            results, chi_sq, minimizer_list = \
-                loop_over_jacobians(controller,
-                                    options,
-                                    grabbed_output)
+            results, minimizer_list = loop_over_jacobians(controller,
+                                                          options,
+                                                          grabbed_output)
 
             for result in results:
-                if problem.multifit:
-                    # Multi fit
-                    # (will raise TypeError if these are not iterable)
-                    for i in range(len(chi_sq)):
-                        result.update({'dataset_id': i,
-                                       'name': '{}, Dataset {}'.format(
-                                           problem.name, (i + 1))})
-                        individual_result = \
-                            fitbm_result.FittingResult(**result)
-                        results_problem.append(individual_result)
-                else:
-                    # Normal fitting
-                    individual_result = fitbm_result.FittingResult(
-                        **result)
-                    results_problem.append(individual_result)
-            new_minimizer_list.extend(minimizer_list)
+                results_problem.append(fitbm_result.FittingResult(**result))
```

Comment:
Is there a reason for keeping the object initialization here rather than moving it with the other code?

Reply:
There was no reason, I'll move it down to where the new code is 👍